assembly/src/release/bin/activemq

```diff
@@ -386,8 +362,8 @@ checkRunning(){
         echo "ERROR: Pidfile '$ACTIVEMQ_PIDFILE' exists but contains no pid"
         return 2
        fi
-       ACTIVEMQ_PID="`cat ${ACTIVEMQ_PIDFILE}`"
-       RET="`ps -p "${ACTIVEMQ_PID}"|grep java`"
+       ACTIVEMQ_PID="`cat $ACTIVEMQ_PIDFILE`"
+       RET="`ps -p "$ACTIVEMQ_PID"|grep java`"
```

Comment:
It's more portable to keep the genere ${xx} format IMHO.

Reply:
from my point of view this doesn't make a huge difference.